lib/grammers-session/src/message_box/mod.rs

```diff
@@ -707,11 +764,14 @@ impl MessageBox {
         request: tl::functions::updates::GetChannelDifference,
         difference: tl::enums::updates::ChannelDifference,
         chat_hashes: &mut ChatHashCache,
-    ) -> (
-        Vec<tl::enums::Update>,
-        Vec<tl::enums::User>,
-        Vec<tl::enums::Chat>,
-    ) {
+    ) -> Result<
```

Comment:
Same here, we always return `Ok` now, so the change to return `Result` is not needed.